### PR TITLE
Update Automation .NET Overview

### PR DESCRIPTION
**Pull request by JeremyLikness**
Added description, examples, and links to samples. Tagging @eamonoreilly and @jwhit-MSFT as engineer/manager and @CamSoper for documents. Someone on the CDA team with review for sign-off.

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-4/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-4/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\tssk-o-unknown\raw-\ogk2wlwt.izq\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\tssk-o-unknown\raw-\ogk2wlwt.izq\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C21%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707211458212853-252%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/automation.md [hunk 1]
```diff
@@ -38,19 +41,44 @@ dotnet add package Microsoft.Azure.Management.Automation
...
+        Parameters = null // optional parameters here
+    }
+};
+
+// create runbook job. This gives back the Job
+Job job = automationManagementClient.Jobs.Create(automationAccountName, jcParam).Job;
+
+string jobId = job.Properties.JobId.ToString();
+
+Console.WriteLine("Job Started for Runbook {0} with Job Id: {1}", runbookName, jobId);
```
Comment (MaximRouiller) [inline api/overview/azure/automation.md]: do we need that in this example?

Comment (MaximRouiller) [inline api/overview/azure/automation.md]: I mean the `Console.WriteLine`

**Comment by MaximRouiller** [comment]
#sign-off

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-4/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/JeremyLikness/azure-docs-sdk-dotnet/blob/patch-4/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\kakp-o-f539a2c3\raw-\zhi3t25z.pe3\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\kakp-o-f539a2c3\raw-\zhi3t25z.pe3\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C21%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201707211517360412-252%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**